### PR TITLE
[German] Add and fix translation

### PR DESCRIPTION
**Pull request by haruzept**
The translation based on the Xiaomi Home app.

### CODE REVIEW

File: src/translations/de.json [hunk 1]
```diff
@@ -10,9 +10,10 @@
   },
   "source": {
     "silent": "Leise",
-    "standard": "Normal",
+    "standard": "Balanciert",
```
Comment (edenhaus) [inline src/translations/de.json:13]: I would prefer to leave it as it is or change it to "Standard"

Comment (denysdovhan) [reply src/translations/de.json:13]: Please, consider this comment.

File: src/translations/de.json [hunk 2]
```diff
@@ -10,9 +10,10 @@
...
     "medium": "Turbo",
-    "turbo": "Max."
+    "turbo": "Max.",
```
Comment (denysdovhan) [inline src/translations/de.json:15]:
```suggestion
    "turbo": "Max",
```

**Review by denysdovhan — CHANGES_REQUESTED** [review]

**Comment by haruzept** [comment]
Is it ok now?

**Review by denysdovhan — APPROVED** [review]

**Pull request merged**